src/server/services/simple-connect/erfpacht.ts

```diff
@@ -444,27 +457,41 @@ export async function fetchErfpachtV2(
 ) {
   const config = getApiConfig('ERFPACHTv2');
 
+  const erfpachterResponse = await requestData<Erfpachtv2ErpachterResponse>(
+    {
+      ...config,
+      url: `${config.url}/vernise/api/erfpachter`,
+      transformResponse: (responseData) =>
+        transformIsErfpachterResponseSource(
+          responseData,
+          authProfileAndToken.profile.profileType
+        ),
+    },
+    requestID,
+    authProfileAndToken
+  );
+
   if (authProfileAndToken.profile.profileType === 'commercial') {
+    return erfpachterResponse;
```

Comment:
is this not already returned at the end of the func? This func then could be terser